fastlane/Fastfile

```diff
@@ -141,8 +144,7 @@ reflected in `Info.plist` during building."
     desc "Define next build number depending on bump_type."
     next_build_number = (bump_type == "build" ? current_build_version.to_i : first_build) + 1
 
-
-    UI.input "next_build_number: #{next_build_number}, class: #{next_build_number.class}"
+    UI.message "next_build_number: #{next_build_number}, class: #{next_build_number.class}"
```

Comment:
This was a log message I was using to debug, it shouldn't be here. To inform what's being uploaded we already have a print in line 189. Erase it.

Reply:
Great! Done.